src/models/h3_neighbor_dataset.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, data: pd.DataFrame):
 
             contexts_indexes = [self.data.index.get_loc(idx) for idx in available_neighbors_h3]
 
-            negative_excluded_h3 = h3.k_ring(h3_index, 1)
+            negative_excluded_h3 = h3.k_ring(h3_index, 2)
```

Comment:
The idea is good. It would make it accurate to the paper (it must have gotten lost somewhere in the process).
I'd suggest making it a parameter of init with a default value of 2, making it a class attribute, and using it here.

Reply:
I like that idea @simonusher. See the latest commit (variable name is a bit long but I wanted to make it explicit)

Reply:
Looks good, thank you.